students/ZackConnaughton/lessons/lesson03/assignment/customer_model.py

```diff
@@ -0,0 +1,44 @@
+"""
+    Simple database examle with Peewee ORM, sqlite and Python
+    Here we define the schema
+    Use logging for messages so they can be turned off
+
+"""
+
+import os
+from peewee import CharField, SqliteDatabase, FloatField, Model, BooleanField
+
+
+DATABASE = SqliteDatabase('customer.db')
+
+def create_database():
+    if not os.path.isfile('customer.db'):
+        DATABASE.connect()
+        DATABASE.execute_sql('PRAGMA foreign_keys = ON;')
+
+        DATABASE.create_tables([
+            Customer
+            ])
+        DATABASE.close()
```

Comment:
Do you really want to do this if customer.db is NOT a file? Don't you want to create only if it is a file?